pkg/apis/componentconfig/edgecore/v1alpha2/default.go

```diff
@@ -54,7 +54,6 @@ func NewDefaultEdgeCoreConfig() *EdgeCoreConfig {
 				TailoredKubeletConfig: &defaultTailedKubeletConfig,
 				TailoredKubeletFlag: TailoredKubeletFlag{
 					HostnameOverride: hostnameOverride,
-					NodeIP:           localIP,
```

Comment:
Also checked, If the `NodeIP` is set, the kubelet‘s setter status will not get by `hostnameOverride`.